CONTRIBUTORS.md

```diff
@@ -191,10 +191,15 @@ List of Contributors
 * [Dick Carter](https://github.com/DickJC123)
 * [Rahul Padmanabhan](https://github.com/rahul3)
 * [Yuxi Hu](https://github.com/yuxihu)
+* [Harsh Patel](https://github.com/harshp8l)
 
 Label Bot
 ---------
 * [mxnet-label-bot](https://github.com/mxnet-label-bot)
-  - mxnet-label-bot provides users with the functionality to add labels from the repository to issues
-  - To use me, comment: @mxnet-label-bot add [specify label here]
+  - mxnet-label-bot provides users with the functionality to manage labels for Issues/Pull Requests on the repository
+  - To use me, comment:
+    - @mxnet-label-bot add [specify comma separated labels here]
+    - @mxnet-label-bot remove [specify comma separated labels here]
+    - @mxnet-label-bot update [specify comma separated labels here] <br>
```

Comment:
why <br>?
You can get a new line without bullet, by using 4 space I guess.

Reply:
Can you add this link for a list of available label names - https://github.com/apache/incubator-mxnet/labels